tests/components/advantage_air/test_climate.py

```diff
@@ -265,3 +269,16 @@ async def test_climate_async_failed_update(
         )
 
     mock_update.assert_called_once()
+
+
+async def test_coordinator_refresh(
+    hass: HomeAssistant,
+    mock_get: AsyncMock,
+    freezer: FrozenDateTimeFactory,
+) -> None:
+    """Test climate after coordinator refresh."""
+
+    await add_mock_config(hass)
+    freezer.tick(ADVANTAGE_AIR_SYNC_INTERVAL)
+    async_fire_time_changed(hass)
+    await hass.async_block_till_done()
```

Comment:
This is testing nothing, this is just touching code

Reply:
Added assertion.